index.html

```diff
@@ -23,25 +23,244 @@ <h1><br>I’m George<br>Glad to see you!</h1>
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <div class="counter"></div>
+            </li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="nav-link">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="see-project">
+            <a href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card1">
+            <img src="./images/Snapshoot-Portfolio3.svg" alt="Card 1" />
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <ul>
+            <li class="upper">Canopy</li>
+            <li>
+              <div class="counter"></div>
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <div class="counter"></div>
+            </li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="nav-link">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="see-project">
+            <a href="#">See Project</a>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <h2>About My Self</h2>
+        <div class="about-content">
+          <p>
+            I’m a software developer! I can help you build a product , feature or
+            website Look through <br>some of my work and experience! If you like what
+            you see and have a project you need<br> coded, don’t hestiate to contact
+            me.
+          </p>
+        </div>
+        <h3 class="connect">LET’S CONNECT</h3>
+        <nav class="nav-icon">
+          <ul>
+            <li>
+              <a href="#"><img src="./images/twitter.svg" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/linkedin.svg" alt="Linkedin" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/github.svg" alt="GitHub" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/angellist.svg" alt="Angellist"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/medium.svg" alt="Medium" /></a>
+            </li>
+          </ul>
+        </nav>
+        <div class="get-my-resume">
+          <p class="button"><a href="#">Get My Resume</a></p>
+        </div>
+        <div class="outside">
+          <a href="#" class="skills-list">
+            <img src="./images/skills-list.svg" alt="List of Skills" />
```

Comment:
- Please remove any horizontal scroll that exists in your design. This is likely caused by an element taking too much space.
![image](https://user-images.githubusercontent.com/24703171/155292417-767635c5-6560-434d-9544-ef9fdb387b3e.png)